modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/ddl/CreateTableCommand.java

```diff
@@ -164,4 +167,20 @@ public void addDataStorageOption(String name, Object value) {
 
         dataStorageOptions.put(name, value);
     }
+
+    /**
+     * Get zone name.
+     */
+    @Nullable
```

Comment:
I don't think that we need Nullable here.

Reply:
Ok, I removed it. But stay it on `public String zone()`.